### PR TITLE
fix: 'Unbury' unavailable on Deck Menu

### LINKED ISSUE #15050: [BUG]: Unbury is not available when enabling v3 scheduler
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

1. Enable new backend and v3 scheduler in settings
2. Start reviewing and bury a card or two
3. Go back to the deck list view
4. Long press the deck with buried cards

### Expected behaviour

There is Unbury item available to unbury the buried cards.

### Actual behaviour

There is no Unbury button in the menu.

### Debug info

```text
AnkiDroid Version = 2.16.5 (953b9bd879269910ee962b520da9705336d8bc2d)

Android Version = 13

ProductFlavor = play

Manufacturer = Sony

Model = XQ-CT72

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; XQ-CT72 Build/64.1.A.0.996; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/120.0.6099.43 Mobile Safari/537.36

ACRA UUID = a43fd0a2-35f4-42e9-a9b1-fca3f77f8102

New schema = true

Scheduler = std3

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

If I switch off v3 scheduler, the Unbury button shows again.

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by david-allison**
## Purpose / Description
* #15050

## Fixes
* Fixes #15050

## Approach
Select the deck + refresh the UI

## How Has This Been Tested?
Unit tested + tested with API 33 emulator

## Learning (optional, can help others)
We need more tests & our alpha testers are fantastic

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/testutils/TestClass.kt [hunk 1]
```diff
@@ -127,6 +129,13 @@ interface TestClass {
     /** Adds [count] notes in the same deck with the same front & back */
     fun addNotes(count: Int): List<Note> = (0..count).map { addNoteUsingBasicModel() }
 
+    /** helper method to update deck config */
+    fun updateDeckConfig(deckId: DeckId, function: DeckConfig.() -> Unit) {
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/testutils/TestClass.kt:133]: Note: this conflicts with the TTS PR

**Review by BrayanDSO — APPROVED** [review]
> & our alpha testers are fantastic

they really are

**Comment by upsuper** [comment]
> our alpha testers are fantastic

I'm not an alpha tester to be honest. I'm using the stable version from play store, and I enabled v3 scheduler manually because the official Anki manual [says](https://faqs.ankiweb.net/the-2021-scheduler.html#compatibility):
> AnkiDroid: 2.16.2+ includes support for the v3 scheduler when the "new backend" option is enabled in the advanced preferences. ... there have been no reports of data loss caused by this option. ...

So I enabled it to keep sync with the PC version, and I found this issue because with v3 scheduler, burying and unburying cards become more important in my review flow.

Thank you for fixing it so quickly!

**Review by mikehardy — APPROVED** [review]
Nice one. Mostly tests!

**Comment by mikehardy** [comment]
@upsuper I just did an alpha so this will probably not see a release until early next week, but 2.17alpha14 will be the one to watch for over here https://github.com/ankidroid/Anki-Android/releases - you can watch the repo for releases only (and then unsubscribe after) if you want github to just let you know without having to check

**Pull request merged**